third_party/cpaembedded/embedded/kiro_account.go

```
@@ -0,0 +1,91 @@
+	// desktop mirror that freecode manages (self-exploration).
+	LoadedViaFreecode bool
+	IncompleteSources []string
+}
+
+// ObserveKiroAccount self-explores the running machine for the Kiro account
+// managed by the Kiro desktop app / AWS SSO cache. Unlike cloud account
+// introspection (which Kiro does not expose publicly), the quota mirror is
+// read directly from the local app state, matching what freecode observes.
+func ObserveKiroAccount(ctx context.Context, credential KiroCredential, options KiroOptions) (KiroAccountObservation, error) {
+	if ctx == nil {
+		ctx = context.Background()
```

golangci-lint (2.13.2)

[error] ineffectual assignment to ctx

(ineffassign)